platform-hub-api/app/models/user.rb

```diff
@@ -21,6 +23,12 @@ class User < ApplicationRecord
 
   has_many :projects, through: :memberships
 
+  has_one :flags,
```

Comment:
A user has one flags object (i.e one record in the `UserFlags` table), as discussed a couple of days ago :)